### PR TITLE
feat: updated to use url redirect location via body (not location header)

### PR DESCRIPTION
**Pull request by madhuravius**
N/A

### CODE REVIEW

File: lib/aptible/cli/helpers/operation.rb [hunk 1]
```diff
@@ -52,8 +52,7 @@ def cancel_operation(operation)
         def operation_logs(operation)
           res = get_operation_logs_redirect(operation)
           # note: res :location is the  header "location" for a 301
```
Comment (joshraker) [inline lib/aptible/cli/helpers/operation.rb]: Comment is no longer applicable

File: lib/aptible/cli/helpers/operation.rb [hunk 2]
```diff
@@ -76,11 +75,10 @@ def get_operation_logs_redirect(operation)
           http = Net::HTTP.new(uri.host, uri.port)
           http.use_ssl = true
           res = http.request(Net::HTTP::Get.new(uri.request_uri, headers))
-          # note: res :location is the  header "location" for a 301
+          # note: res body with a 200 is target redirect location for download
           # in approx ruby 2.4 or lower, location is in a different spot
```
Comment (joshraker) [inline lib/aptible/cli/helpers/operation.rb]: Comment is no longer applicable

File: lib/aptible/cli/helpers/operation.rb (hunk 1 above)
Comment (madhuravius) [reply lib/aptible/cli/helpers/operation.rb]: dropped this one

File: lib/aptible/cli/helpers/operation.rb (hunk 2 above)
Comment (madhuravius) [reply lib/aptible/cli/helpers/operation.rb]: dropping the 2nd comment, but i updated the first i think when this was reviewed to be somewhat accurate

**Pull request merged**